SparkPost/matchbox
# UX-429 Add appearance prop on LabelValue component and document

### What Changed

- new `appearance` prop (accepts `"inverted"`) to invert the color scheme of the `LabelValue` component for light text on dark background
- Documents new `appearance` prop on design.sparkpost.com

### How To Test or Verify

- `npm run start:storybook`
- Verify new [appearance story](http://localhost:9001/?path=/story/layout-labelvalue--inverted-appearance)
### PR Checklist
